src/Servers/Kestrel/Core/src/Internal/Certificates/CertificateConfigLoader.cs

```diff
@@ -37,6 +37,14 @@ public CertificateConfigLoader(IHostEnvironment hostEnvironment, ILogger<Kestrel
         else if (certInfo.IsFileCert)
         {
             var certificatePath = Path.Combine(HostEnvironment.ContentRootPath, certInfo.Path!);
+
+            X509Certificate2Collection? fullChain = null;
+            if (certInfo.Path != null)
```

Comment:
`certInfo.IsFileCert` already verifies this.

Reply:
Cool so I guess we can always build this full chain then